srml/contract/src/lib.rs

```diff
@@ -73,17 +73,18 @@ use rstd::marker::PhantomData;
 use parity_codec::{Codec, Encode, Decode};
 use runtime_primitives::traits::{Hash, As, SimpleArithmetic,Bounded, StaticLookup};
 use srml_support::dispatch::{Result, Dispatchable};
-use srml_support::{Parameter, StorageMap, StorageValue, StorageDoubleMap, decl_module, decl_event, decl_storage};
+use srml_support::{Parameter, StorageMap, StorageValue, StorageDoubleMap, decl_module, decl_event, decl_storage, storage::child};
 use srml_support::traits::OnFreeBalanceZero;
 use system::{ensure_signed, RawOrigin};
 use runtime_io::{blake2_256, twox_128};
 use timestamp;
 use fees;
 
 pub type CodeHash<T> = <T as system::Trait>::Hash;
+pub type KeySpace = Vec<u8>;
```

Comment:
I find the name `KeySpace` to be confusing, tbh. I remember you told me, that you chose that name because of some underlying details of the storage implementation. However, I think that the connection is somewhat convoluted.

I'd prefer leave the name `trie_id` as found in the original design doc. This would involve changing the other names as well.

Reply:
As in my comment bellow, I ended up to similar conclusion (keyspace being inplementation details of child trie).